tests/subjective_billing_test.py

```diff
@@ -0,0 +1,199 @@
+
+    preBalances = node.getEosBalances([account1, account2])
+    Print("Starting balances:")
+    Print(preBalances)
+
+    #   Send through some failing transactions on 30 min decay node
+    for x in range(5):
+        memo = 'tx-{}'.format(x)
+        txn = {
+
+            "actions": [{"account": "eosio.token","name": "transfer",
+                         "authorization": [{"actor": "account1","permission": "active"}],
+                         "data": {"from": "account1","to": "account2","quantity": "100000.0001 SYS","memo": memo},
+                         "compression": "none"}]
+        }
+
+        results = fdnode.pushTransaction(txn)
+        assert(not results[0])
+
+    # Get subjective billing.  We'll check on this later.
+    decaySubjectiveBilling = fdnode.getAccountSubjectiveInfo("account1")
+    Print('Subjective billing used from fast decay node {}'.format(decaySubjectiveBilling["used"]))
+
+
+    #   Send through some failing transactions on a non-whitelisted account
+    Print('Verifying non-whitelisted account...')
+    subjectiveBilling = wlnode.getAccountSubjectiveInfo("account2")
+    Print('Initial Subjective billing used: {}'.format(subjectiveBilling["used"]))
+
+    for x in range(5):
+        memo = 'tx-{}'.format(x)
+        txn = {
+
+            "actions": [{"account": "eosio.token","name": "transfer",
+                         "authorization": [{"actor": "account2","permission": "active"}],
+                         "data": {"from": "account2","to": "account1","quantity": "100000.0001 SYS","memo": memo},
+                         "compression": "none"}]
+        }
+
+        results = wlnode.pushTransaction(txn)
+        assert(not results[0])
+
+    # Verify that no subjective billing was charged
+    subjectiveBilling = wlnode.getAccountSubjectiveInfo("account2")
+    Print(subjectiveBilling)
+    Print(subjectiveBilling["used"])
+    assert(subjectiveBilling["used"] > 0)
+
+    #   Send through some failing transactions on a whitelisted account
+    Print("Verifying whitelisted account...")
+    subjectiveBilling = wlnode.getAccountSubjectiveInfo("account1")
+    Print('Initial Subjective billing used: {}'.format(subjectiveBilling["used"]))
+    for x in range(5):
+        memo = 'tx-{}'.format(x)
+        txn = {
+
+            "actions": [{"account": "eosio.token","name": "transfer",
+                         "authorization": [{"actor": "account1","permission": "active"}],
+                         "data": {"from": "account1","to": "account2","quantity": "100000.0001 SYS","memo": memo},
+                         "compression": "none"}]
+        }
+
+        results = wlnode.pushTransaction(txn)
+        assert(not results[0])
+
+    # Verify that no subjective billing was not charged
+    subjectiveBilling = wlnode.getAccountSubjectiveInfo("account1")
+    Print(subjectiveBilling["used"])
+    assert(subjectiveBilling["used"] == 0)
+
+
+    # Sleep for 5 min
+    time.sleep(300)
```

Comment:
Can we set decay lower and reduce this sleep. 5 minutes is a long time for a test to just sit.